src/carplay.cs

```diff
@@ -657,10 +905,12 @@ interface ICPSearchTemplateDelegate { }
 	interface CPSearchTemplateDelegate {
 
 		[Abstract]
+		[Async]
```

Comment:
Same, I don't think this is async:

> After the method retrieves the search result, the method must call completionHandler for the system to display the result to the user.

Reference: https://developer.apple.com/documentation/carplay/cpsearchtemplatedelegate/2968421-searchtemplate?language=objc